simulation_parameters/microbe_stage/organelles.json

```diff
@@ -719,6 +719,40 @@
     "editorButtonGroup": "External",
     "editorButtonOrder": 3
   },
+  "slimeJet": {
+    "mpCost": 60,
+    "initialComposition": {
+      "ammonia": 1,
+      "phosphates": 1
+    },
+    "hexes": [
+      {
+        "r": 0,
+        "q": 0
+      }
+    ],
+    "processes": {
+      "mucilage_synthesis": 1
+    },
+    "components": {
+      "processor": {},
+      "storage": {
+        "capacity": 0.5
+      },
+      "slimeJet": {}
+    },
+    "prokaryoteChance": 0.3,
+    "chanceToCreate": 0.3,
+    "mass": 0.1,
+    "displayScene": "res://assets/models/organelles/SlimeJet.tscn",
+    "displaySceneModelPath": "Armature/Skeleton/Cube",
+    "displaySceneAnimation": "AnimationPlayer",
+    "name": "SLIME_JET",
+    "consumptionColour": "#00ffcc",
+    "iconPath": "res://assets/textures/gui/bevel/parts/SlimeJetIcon.png",
+    "editorButtonGroup": "External",
+    "editorButtonOrder": 3
```

Comment:
This number needs to be unique in the group, I can see in the github diff that the number 3 is already in use by the organelle above this one.